Privacy.md

```diff
@@ -0,0 +1,88 @@
+# Forecastie
+
+# Privacy Policy
+
+This privacy policy statement is created to clearly state the developer's policies with respect to data collection, usage, and security. These policies are stated to ensure you know what personal details you may or may not disclose when the software is being utilized, and the choices you have associated with that data.
+If you choose to use this Service, then you agree to the collection and use of information in relation to this policy. The Personal Information that may be collected is used for providing and improving the Service. This information will not be used or shared with anyone as described in this Privacy Policy.
+Kindly note that this Privacy Policy applies only to this software. The developer would not be held responsible for the content or privacy practices on any third party app not operated by the developer to which this software links, or that links to this software. Therefore, it is advisable to read up the privacy policies of third party services that may be linked to this software.
+
+## Permission and Access
+
+For a better experience, while using our Service, I may require you to provide us with certain personally identifiable information, and the enablement of certain device permission. This permission needs to be enabled for the software to work properly on the device on which the software's services are intended. The information that the developer requests with regards to these permissions will be retained on your device and is not collected by us in any way.
+
+Forecastie needs this permission below;
+
+- Location
+
+With the location permission enabled, the app can;
+
+- access approximate location (network-based).
+- access precise location (GPS and network-based).
+
+Forecastie has some other capabilities, which are functions the software can remotely effect on its own. They include;
+
+- **have full network access:** The app is capable of creating network sockets and can use customized network protocols. This capability does not require the software to send data to the internet.
+- **view network connections:** This is the capability of the app to view information about network connections such as which networks exist and are connected.
+- **run at startup:** Allows the app to have itself started as soon as your device has finished booting.
+
+## Types of Data Collected
+
+### Personal Data
+The developer doesn't collect personally identifiable data.
+
+### Usage Data
+The developer doesn't collect usage data.
+
+### Installation Data
+The developer distributes this software via channels like F-Droid and GitHub where download counts are not maintained. So the developer has no knowledge or way to acquire knowledge that the user has installed or is using this software. The developer also distributes this software via google play store, but the developer is not privy to this information either.
```

Comment:
Download counts are listed at Google Play, making this claim false. The sentence reads as if the dev doesn't know about distribution to Google Play.
```suggestion
The developer distributes this software via channels like F-Droid and GitHub where download counts are not maintained. So the developer has no knowledge or way to acquire knowledge that the user has installed or is using this software. The developer also distributes this software via Google Play, but the developer is not privy to this information either.
```